pymc3/step_methods/hmc/base_hmc.py

```diff
@@ -201,9 +201,9 @@ def astep(self, q0):
         stats = {
             "tune": self.tune,
             "diverging": bool(hmc_step.divergence_info),
-            "perf_counter_diff_ns": perf_end - perf_start,
-            "process_time_diff_ns": process_end - process_start,
-            "perf_counter_ns": perf_end,
+            "perf_counter_diff": perf_end - perf_start,
```

Comment:
Just a suggestion: why not call this `perf_time_diff`? I find it more explicit and it matches `process_time_diff`

Reply:
I see why `perf_time_diff` might be nicer, I just followed the naming of the function in `time`, so that it's easier to see what clock is used exactly. But `counter` is a bit confusing...